### DISCUSSION TITLE
[2024-12-06] Meeting minutes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1675] [Weekly Meetings]
- Apache KIE 10.0.0
   - Community Vote passed!
      - https://lists.apache.org/thread/lnrx61q01n8pszqymcfqw90cq4r6mmdq
   - IPMC Vote thread ongoing
      - https://lists.apache.org/thread/s5sqz2nob8sqmr0gs8qnthp6gt0r36vp
- GWT related code
   - Legal ticket opened - https://issues.apache.org/jira/browse/LEGAL-691
- Migrating from jUnit 4 to 5
   - Simulator for scenario... the runner has changed and became very different.
   - Is this module still relevant?
      - [@porcelli ] it's relevant to Alex perspective
      - [@pibizza] it'll likely require investigation to understand how to make it compatible to junit 5
      
----
Recording 

https://us06web.zoom.us/rec/share/R87XsgsC2GtHeswzqjaYpl4DkHsq63Kqoz-4ToLN-lzT9HkHaUHv4hZY5ZwHrjKc.-a_-ik7-Idts4hOe?startTime=1733493563000
Passcode: ^s.08&av

**Comment by porcelli** [comment]
@yesamer @gitgabrio you may be interested in the topic of Migrating from jUnit 4 to 5

**Reply by gitgabrio** [reply]
Yes @porcelli @pibizza 
That scenario simulation thing should be kept up and running: used a lot (AFAIK) and extremely useful
I'll take a look ASAP.

**Reply by yesamer** [reply]
Thank you @gitgabrio for opening a ticket https://github.com/apache/incubator-kie-issues/issues/1678 to track this topic.
